WhaleSpotting/Controllers/TripPlannerController.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Models.Response;
+using WhaleSpotting.Services;
+using WhaleSpotting.Models.Request;
+
+namespace WhaleSpotting.Controllers;
+
+[ApiController]
+[Route("plan-trip")]
+public class TripPlannerController : ControllerBase
+{
+    private readonly ITripPlannerService _tripPlannerService;
+
+    public TripPlannerController(ITripPlannerService tripPlannerService)
+    {
+        _tripPlannerService = tripPlannerService;
+    }
+
+    [HttpGet("")]
+    public ActionResult<List<TripPlannerResponse>> GetNearBySightings([FromQuery] TripPlannerRequest input)
+    {
+        try
+        {
+            var response = _tripPlannerService.ListNearBySightings(input.lat, input.lon);
+            return response;
```

Comment:
Can we rename this variable to be more self-explanatory?
```suggestion
            var topFiveSightings = _tripPlannerService.ListNearBySightings(input.lat, input.lon);
            return topFiveSightings;
```

Reply:
changed